src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -1,1233 +1,1265 @@
+        for comp, s in _mf_scale.items():
+            self.o2_side_prop_params.set_default_scaling(
+                "mole_frac_comp", s, index=comp
+            )
+            self.o2_side_prop_params.set_default_scaling(
+                "mole_frac_phase_comp", s, index=comp
+            )
+
+        self.h2_side_prop_params.set_default_scaling("mole_frac_comp", 10)
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_phase_comp", 10)
+
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_comp", 1)
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_phase_comp", 1)
+
+    def _define_cell_params(self):
+        self.soec_module.number_cells.fix(1e6)#1.8*4.4e5)
+
+        self.soec_module.solid_oxide_cell.fuel_channel.length_x.fix(0.002)
+        self.soec_module.solid_oxide_cell.length_y.fix(0.2345)
+        self.soec_module.solid_oxide_cell.length_z.fix(0.2345)
+        self.soec_module.solid_oxide_cell.fuel_channel.heat_transfer_coefficient.fix(100)
+
+        self.soec_module.solid_oxide_cell.oxygen_channel.length_x.fix(0.002)
+        self.soec_module.solid_oxide_cell.oxygen_channel.heat_transfer_coefficient.fix(100)
+
+        self.soec_module.solid_oxide_cell.fuel_electrode.length_x.fix(1e-3)
+        self.soec_module.solid_oxide_cell.fuel_electrode.porosity.fix(0.326)
+        self.soec_module.solid_oxide_cell.fuel_electrode.tortuosity.fix(3)  # Revisit
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_heat_capacity.fix(595)
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_density.fix(7740.0)
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_thermal_conductivity.fix(6.23)
+        self.soec_module.solid_oxide_cell.fuel_electrode.resistivity_log_preexponential_factor.fix(
+            pyo.log(2.5e-5)
+        )
+        self.soec_module.solid_oxide_cell.fuel_electrode.resistivity_thermal_exponent_dividend.fix(0)
+
+        self.soec_module.solid_oxide_cell.oxygen_electrode.length_x.fix(40e-6)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.porosity.fix(0.30717)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.tortuosity.fix(3.0)  # Revisit
+        # Heat capacity and heat transfer coefficients of oxygen electrode aren't well known but probably don't matter
+        # because the electrode is extremely thin
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_heat_capacity.fix(142.3)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_density.fix(3030)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_thermal_conductivity.fix(5.84)
+        # Also unknown but probably insignificant
+        self.soec_module.solid_oxide_cell.oxygen_electrode.resistivity_log_preexponential_factor.fix(
+            pyo.log(7.8125e-05)
+        )
+        self.soec_module.solid_oxide_cell.oxygen_electrode.resistivity_thermal_exponent_dividend.fix(0)

```

Comment:
I had to update the `resistivity_thermal_exponent_dividend` value to `1e-8` to get the soec model to converge.

Reply:
I'm not sure.  Mine worked, but seems like the change would be fine.  @dallan-keylogic any thoughts on this?

Reply:
@chineduobiora  What sort of problem was occurring? Why did you choose to update this particular value?

The change itself shouldn't be a big deal, but it's not clear to me why it should be necessary.